src/appleseed/renderer/modeling/light/light.h

```diff
@@ -79,18 +79,25 @@ class APPLESEED_DLLSYMBOL Light
 
     enum Flags
     {
-        CastIndirectLight = 1 << 0                                  // does this light generate indirect lighting?
+        CastIndirectLight = 1 << 0,         // does this light generate indirect lighting?
+        LightTreeCompatible = 1 << 1        // can this light be used by the LightTree?
     };
 
     // Retrieve the flags.
     int get_flags() const;
 
+    // Set the flags
+    void set_flags(Flags flag);
+
     // Retrieve the importance multiplier.
     float get_uncached_importance_multiplier() const;
 
     // Set the light transformation.
     void set_transform(const foundation::Transformd& transform);
 
+    // Set the flag that the light is LightTreeCompatible
+    bool is_light_tree_compatible() const;
```

Comment:
The comment is wrong indeed, the method is still used in `Light::on_frame_begin()` to prevent the flag from being overwritten. I think we should keep it for now as it makes the `if` there more understandable